File: tests/ReservedMemberAccess.cs
namespace Demo {
    public partial class Tnulo {
        public int fInt;
        public int @int { get => fInt; set => fInt = value; }
    }
    
    public partial class TUse {
        public void DoIt(Tnulo n) {
            n.@int = 5;
        }
    }
}

File: tests/ReservedMemberAccess.pas
namespace Demo;

type
  Tnulo = public class
  private
    fInt: Integer;
  public
    property int: Integer read fInt write fInt;
  end;

  TUse = public class
  public
    method DoIt(n: Tnulo);
  end;

implementation

method TUse.DoIt(n: Tnulo);
begin
  n.int := 5;
end;

end.
